src/Common/tests/Scripts/Tools/ParallelTestExecution.ps1

```diff
@@ -80,7 +80,7 @@ function ParseTestExecutionCommand($msBuildOutput)
     }
 
     $Global:TestPath = $pathLine.Trim()
-    $Global:TestCommand = $cmdLine.Trim() -replace ("call ", ".\")
```

Comment:
The script is parsing the output to figure out the command that executes a single instance of the test.
A recent test change breaks the current output parser: the command it executes the test isn't shown anymore... This means that the script is broken yet again. 

Removing this change from the PR.